src/components/Toolbelt/Toolbelt.js

```diff
@@ -20,17 +20,15 @@ import './Toolbelt.sass'
 const noop = () => {}
 
 const getTools = memoize(toolLevels => {
-  let tools = []
+  const tools = []
 
   for (let tool of Object.values(toolsData)) {
     if (toolLevels[tool.type] !== toolLevel.UNAVAILABLE) {
       tools.push(tool)
     }
   }
 
-  tools.sort(t => t.order)
```

Comment:
This was actually broken previously. I'm not sure when; I think I introduced this bug. The broken sorting was breaking keyboard combos, so this will be nice to ship a fix for!